lib/filter.g

```diff
@@ -344,4 +345,70 @@ BIND_GLOBAL( "CANONICAL_BASIS_FLAGS", QUO_INT(SUM_FLAGS,5) );
 
 #############################################################################
 ##
+#V  IdOfFilterByName
+#V  FilterByName
+##
+##  <#GAPDoc Label="FilterByName">
+##  <ManSection>
+##  <Func Name="IdOfFilterByName" Arg="name"/>
+##  <Func Name="FilterByName" Arg="name"/>
+##
+##  <Description>
+##  finds the id of the filter, or the filter with name <A>name</A> respectively,
```

Comment:
My first thought was "what is the id of a filter"? I only later saw `IdOfFilter` below. I'd therefore recommend splitting the documentation of `IdOfFilterByName` and `FilterByName`. The former might start like this:
"find the id of the given filter, as defined in `IdOfFilter`.